perfact/zodbsync/subcommand.py

```diff
@@ -73,6 +75,81 @@ def __init__(self, args):
         self.sync = ZODBSync(conffile=args.config, logger=logger)
         self.config = self.sync.config
 
+    def gitcmd(self, *args):
+        return ['git', '-C', self.sync.base_dir] + list(args)
+
+    def gitcmd_run(self, *args):
+        '''Wrapper to run a git command.'''
+        subprocess.check_call(self.gitcmd(*args))
+
+    def gitcmd_output(self, *args):
+        '''Wrapper to run a git command and return the output.'''
+        return subprocess.check_output(
+            self.gitcmd(*args), universal_newlines=True
+        )
+
+    def check_path(self, path):
+        '''Check a given path to be relative (assume its a Data.fs path) or
+        an absolute path (assume its a filesystem path). Return both.
```

Comment:
I suggest a different approach: If a path starts with a dot, it is assumed to be a relative file system path. I usually will have no need to specify a file system path absolutely (by default, this would require typing out `/opt/perfact/dbutils-zoperepo/__root__/...`). However, I might be located in `/opt/perfact/dbutils-zoperepo/__root__/PerFact/DB_Utils/lib` and want to upload a folder to the path where I am - in that case, I could start with a dot and use shell completion to write `zodbsync upload ~/jquery_js ./jquery_js` to upload a library to a path I want to specify.

Also, other commands already use and document the use of paths starting with a `/` meaning paths inside the Data.fs, so drawing the distinction between absolute and relative paths would create an inconsistent pattern.